### ISSUE TITLE
Updated FlashBlade collection in Galaxy

### ISSUE THREAD

**Issue by rickhlx** [Pure-Storage-Ansible/FlashBlade-Collection#293]
I notice there's not been a published collection update to FlashBlade Galaxy collection since April. 

I've run into a bug and would like to update my collection via Galaxy. 

Would it be possible to publish an up-to-date collection to Galaxy?

**Comment by sdodsley** [comment]
@rickhlx We are waiting on 1 custmer issue to be confirmed as resolved and then we will release 1.18.0

**Comment by sdodsley** [comment]
No - that is not possible.

What is the bug you are experiencing? We should check it has been fixed for the next release.
Can you raise a separate issue describing the bug so we can confirm it is already fixed, or if we need to create a new patch to roll into the next release?
